marlowe-playground-client/static/css/panels.scss

```diff
@@ -263,29 +262,37 @@ button.minus-btn:hover {
   outline: none;
 }
 
-.metadata-field-group {
-  margin: 0.5rem;
-}
-
-.metadata-field-group h6 {
-  margin-top: 1rem;
-  margin-bottom: 0.5rem;
+.metadata-group-title h6 {
   font-weight: bold;
-}
-
-.metadata-field-group li {
-  margin-left: 1rem;
-}
-
-.metadata-field-group li::marker {
-  display: none;
-  content: " ";
+  margin: 0.5rem 0;
 }
 
 .metadata-error {
   color: var(--error-color);
 }
 
 .metadata-not-used {
-  margin-left: 1rem;
-}
\ No newline at end of file
+  row-gap: 1rem;
+}
+
+.metadata-form {
+  display: grid;
+  grid-template-columns: auto 1fr auto auto;
+  gap: 0.5rem 0px;
+  width: 100%;
+  box-sizing: border-box;
+  align-items: center;
+  max-width: 75rem;
+  margin: auto;
```

Comment:
I'd remove `margin: auto`. I don't think we want to align the contents in the center of the panel.

![Screen Shot 2021-03-15 at 11 02 57](https://user-images.githubusercontent.com/2634059/111165898-4f6f4d80-857e-11eb-9b9d-d31391fff958.png)

![Screen Shot 2021-03-15 at 11 03 15](https://user-images.githubusercontent.com/2634059/111165920-56965b80-857e-11eb-9f59-6086fedbcfac.png)

Reply:
Maybe I should make it 100% width, but I think aligning it left doesn't look very good either :/

Reply:
![Screenshot_20210315_142757](https://user-images.githubusercontent.com/638102/111169370-ca466180-859a-11eb-981f-a871fa415326.png)

Reply:
or max-width 80% 🤔